RadarSDKTests/RadarSDKTests.m

```diff
@@ -823,7 +823,7 @@ - (void)test_Radar_getContext_errorLocation {
         [expectation fulfill];
     }];
 
-    [self waitForExpectationsWithTimeout:60
+    [self waitForExpectationsWithTimeout:120
```

Comment:
What's the explanation for why we weren't seeing this before? Or were we?

Reply:
we were, they come sporadically after we upgraded the github action to simulate an iphone 15 pro